src/main/resources/static/css/buttonOverrides.css

```diff
@@ -1,3 +1,25 @@
+.button_disabled
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename selector "button_disabled" to match the regular expression: ^[a-z][-a-z0-9]*$ [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=css%3Aselector-naming-convention)